Setup express server

adrianhumphrey111 opened the pull request:
Simple working express server running on localhost:8080.

lamdbui committed:
Add contribution guidelines with addtional develop branch

lamdbui committed:
Add contribution guidelines with addtional develop branch

lamdbui committed:
Add additional info for pushing branch to remote

lamdbui committed:
Add additional info for pushing branch to remote

leungleoqin committed:
Create database schema and dummy data

lamdbui committed:
Fix typo

adrianhumphrey111 committed:
Set up a simple working server on localhost:8080 while also implementing socket.io.

adrianhumphrey111 committed:
Set up a simple working server on localhost:8080 while also implementing socket.io.

adrianhumphrey111 committed:
Added package-lock.json in order to make sure eveyone is using the correct version of modules.

adrianhumphrey111 committed:
Added package-lock.json in order to make sure eveyone is using the correct version of modules.

lamdbui reviewed (changes requested):
The basic express portion looks fine to me. I'll let @leungleoqin comment on the socket stuff.

Also, on the pull request set the branch to merge into from 'master' to 'develop'

leungleoqin reviewed (changes requested):
We discussed today that we should have two separate routes, '/' for the Game Host (page to be shown on projector), and '/join' for individual Game Players (page to be shown on phone screen). As for the socket communication, I am working on the event names now, but we probably don't need to include those until we start writing the server code.

leungleoqin committed:
List out all socket events between server, game host and game player

lamdbui committed:
Add initial guidelines for AirBnB style and linting

lamdbui committed:
Add initial README details

adrianhumphrey111 committed:
Added a /join route for clients to visit in order to join the current game.

adrianhumphrey111 committed:
Added a /join route for clients to visit in order to join the current game.

Commit:
Merge branch 'develop' into setup-express-server

leungleoqin committed:
Pull from setup-express-server

Merge branch 'setup-express-server' of https://github.com/outergamespace/outergamespace into setup-express-server

leungleoqin committed:
Merge branch 'develop' into setup-express-server
